SU2_CFD/src/fluid/CNEMOGas.cpp

```diff
@@ -0,0 +1,110 @@
+/*!
+ * \file CNEMOGas.cpp
+ * \brief Source of the nonequilibrium gas model.
+ * \author W. Maier, C. Garbacz
+ * \version 7.0.5 "Blackbird"
+ *
+ * SU2 Project Website: https://su2code.github.io
+ *
+ * The SU2 Project is maintained by the SU2 Foundation
+ * (http://su2foundation.org)
+ *
+ * Copyright 2012-2020, SU2 Contributors (cf. AUTHORS.md)
+ *
+ * SU2 is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * SU2 is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with SU2. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "../../include/fluid/CNEMOGas.hpp"
+
+CNEMOGas::CNEMOGas(const CConfig* config): CFluidModel(){
+
+  nSpecies            = config->GetnSpecies();
+  MassFrac_Freestream = config->GetMassFrac_FreeStream();
+
+  MolarMass.resize(nSpecies,0.0);
+  MolarFractions.resize(nSpecies,0.0);
+  Cvtrs.resize(nSpecies,0.0);         
+  Cvves.resize(nSpecies,0.0);               
+  eves.resize(nSpecies,0.0);            
+  hs.resize(nSpecies,0.0);            
+  ws.resize(nSpecies,0.0);            
+  DiffusionCoeff.resize(nSpecies,0.0);
+  temperatures.resize(nEnergyEq,0.0);
+  energies.resize(nEnergyEq,0.0);  
+  ThermalConductivities.resize(nEnergyEq,0.0);
+
+  Kind_TransCoeffModel = config->GetKind_TransCoeffModel();
+
+}
+
+CNEMOGas::~CNEMOGas(){}
+
```

Comment:
If you don't need to delete anything explicitly it is best to not even declare the destructor.